itests/kit/ensemble_opts.go

```diff
@@ -65,3 +65,10 @@ func Account(key *key.Key, balance abi.TokenAmount) EnsembleOpt {
 		return nil
 	}
 }
+
+func Time(offset time.Duration) EnsembleOpt {
```

Comment:
I'm actually going to delete this because I ended up hacking around itest timing in a different way.  Using this option makes tests incredibly slow so its kind of a footgun